src/scene_battle_rpg2k.cpp

```diff
@@ -467,52 +517,31 @@ bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase
 				Game_System::SePlay(*action->GetStartSe());
 			}
 
-			battle_action_state = BattleActionState_Result;
+			action->Apply();
```

Comment:
The moving down of the Apply() call should make no difference as the code before doesn't depend on the side-effects of Apply